### DISCUSSION TITLE
Referencing equation numbers with brakets

### DISCUSSION THREAD

**Discussion by gdietl** [quarto-dev/quarto-cli#3354] [Q&A]
When referencing equations, the brakets around equation number are missing. Is there a way to change this behaviour?

**Comment by mcanouil** [comment]
Could you provide a working reproducible example?

Is it related to #2413?

**Comment by gdietl** [comment]
Exactly, it is the same problem as described in #2413. Sorry that I did not find it before although I've searched for it.

Sample code:
```
---
title: "My Document"
format: 
  html:
    toc: true
    code-fold: true
---

# This is a sample document

$$\hat{H}\psi = E\psi$$ {#eq-schroedinger}

Please refer to @eq-schroedinger.
```
Result:
![Screenshot_20221117_092122](https://user-images.githubusercontent.com/112486210/202393905-952b3413-6afc-4a2a-ba6f-d550a3e7f44c.png)

**Reply by mcanouil** [reply]
Then I believe @cderv answer in #2413 is still the current answer to your question.

**Reply by gdietl** [reply]
Thanks. I've read this answer but this workaround doesn't help me because I am interested in the HTML output. Do you know if someone has filed a feature request?

**Reply by mcanouil** [reply]
A Lua filter parsing links and searching/replacing for pattern should work, i.e., `<a href="#eq-schroedinger">Equation&nbsp;1</a>`.
(I am not a Lua expert, I'll try later to make one if a Lua filter is not provided by then).

**Comment by cderv** [comment]
There is an open issue about this topic https://github.com/quarto-dev/quarto-cli/issues/2439

You should follow there for updates and add anything missing in the issue. But in short, we are aware of the limitation and we'll try to work on it for 1.3 as the issue milestone state
